<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingSystem</class>
 <widget class="QMainWindow" name="TestingSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestingSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="settingsMenuBar">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="openSettingsMenu"/>
   </widget>
   <widget class="QMenu" name="infoMenuBar">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="testingRulesMenu"/>
    <addaction name="editQuestionsRulesMenu"/>
    <addaction name="statMenu"/>
   </widget>
   <addaction name="settingsMenuBar"/>
   <addaction name="infoMenuBar"/>
  </widget>
  <action name="openSettingsMenu">
   <property name="text">
    <string>Настрокйи тестирования</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="pathesToMaterialMenu">
   <property name="text">
    <string>Расположение материала</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="testingRulesMenu">
   <property name="text">
    <string>Правила тестирования</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="editQuestionsRulesMenu">
   <property name="text">
    <string>Редактирование вопросов</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="statMenu">
   <property name="text">
    <string>Статистика</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
